src/modules/handlers/hello.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+async function hello(ctx) {
+  try {
```

Comment:
Why `try catch` here ?

Reply:
In this case it's useless to use try catch,  in other cases it may be usefull. I will remove it.